chatcommands.py

```diff
@@ -194,29 +194,32 @@ def blacklist(_):
     Returns a string which explains the usage of the new blacklist commands.
     :return: A string
     """
-    raise CmdException("The !!/blacklist command has been deprecated. "
-                       "Please use !!/blacklist-website, !!/blacklist-username,"
-                       "!!/blacklist-keyword, or perhaps !!/watch-keyword. "
-                       "Remember to escape dots in URLs using \\.")
+    raise CmdException("The `!!/blacklist` command has been deprecated. "
+                       "Please use `!!/blacklist-website`, `!!/blacklist-username`, "
+                       "`!!/blacklist-keyword`, or perhaps `!!/watch`. "
+                       "Remember to escape dots in URLs using `\\.`, "
+                       "and spaces and punctuations with `\\W?`.")
 
 
-def check_blacklist(string_to_test, is_username, is_watchlist):
+def check_blacklist(string_to_test, **is_what):
```

Comment:
`**is_a`?

Reply:
You really care about naming convention? I think the current name `is_what` is just fine. Anyway it's trivial to change that.